### ISSUE TITLE
Unable to manage services on systems relying on update-rc.d

### ISSUE THREAD

**Issue by dennisconrad** [ansible/ansible#42734]
##### SUMMARY

The new `sysvinit` module in ansible 2.6 has a typo, which renders the system unable to detect the `update-rc.d` executable:

```python
    # locate binaries for service management
    paths = ['/sbin', '/usr/sbin', '/bin', '/usr/bin']
    binaries = ['chkconfig', ' update-rc.d', 'insserv', 'service']
```
Note the leading space in `' update-rc.d'`.  This breaks the ability to enable / disable services on systems relying on `update-rc.d` for it, and breaks idempotency for `state=enabled` on such systems with the service disabled, due to the implied `enabled=yes` in that case.

##### ISSUE TYPE
 - Bug Report

##### COMPONENT NAME
sysvinit

##### ANSIBLE VERSION

```
ansible 2.6.1
  config file = None
  configured module search path = [u'/Users/dennis.conrad/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /Users/dennis.conrad/github/sansible/elasticsearch/.venv_ansible2.6.1/lib/python2.7/site-packages/ansible
  executable location = /Users/dennis.conrad/github/sansible/elasticsearch/.venv_ansible2.6.1/bin/ansible
  python version = 2.7.14 (default, Jun 18 2018, 15:53:27) [GCC 4.2.1 Compatible Apple LLVM 9.0.0 (clang-900.0.39.2)]
```

##### OS / ENVIRONMENT
All systems relying on `update-rc.d` to manage services.

##### STEPS TO REPRODUCE

Install, but disable, a service on a system relying on `update-rc.d` to manage service (e.g. Ubuntu Bionic), then try to enable the service using the following snippet:

```yaml
- name: Ensure Service XXX is Enabled
  become: yes
  service:
    name: XXX
    enabled: yes
```

##### EXPECTED RESULTS

The service is enabled, and the task returns `changed=true`.

##### ACTUAL RESULTS

The service is *NOT* enabled, yet the task still returns `changed=true`.

**Comment by dennisconrad** [comment]
Added PR to fix issue https://github.com/ansible/ansible/pull/42735 .

**Comment by bcoca** [comment]
closed via #42735
